packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5882,7 +5901,57 @@ export default class Meeting extends StatelessWebexPlugin {
       throw error;
     }
 
-    await this.waitForMediaConnectionConnected();
+    try {
+      await this.waitForMediaConnectionConnected();
+
+      if (this.retriedWithTurnServer) {
+        Metrics.sendBehavioralMetric(
+          BEHAVIORAL_METRICS.RETRY_MEDIA_CONNECTION_WITH_TURN_SERVER_SUCCESS,
+          {
+            correlation_id: this.correlationId,
+          }
+        );
+      }
+    } catch (error) {
+      // If waitForMediaConnectionConnected() fails when we haven't done turn discovery (and the config allows for turn discovery to happen) then we
+      // attempt to establish a media connection again but this time using turn discovery. If we used turn discovery on the first pass we do not attempt
+      // connection again.
+
+      // @ts-ignore - config coming from registerPlugin
+      if (!this.turnServerUsed && this.turnDiscoverySkippedReason !== 'config') {
+        LoggerProxy.logger.error(
+          `${LOG_HEADER} error waiting for media to connect on UDP/TCP, retrying using TURN server, `,
+          error
+        );
+        this.retriedWithTurnServer = true;
+
+        await this.cleanUpBeforeRetryWithTurnServer();
+
+        try {
+          await this.establishMediaConnection(remoteMediaManagerConfig, bundlePolicy, true);
+        } catch (err) {
+          LoggerProxy.logger.error(
+            `${LOG_HEADER} retry with TURN server failed, media connection unable to connect, `,
+            err
+          );
+          Metrics.sendBehavioralMetric(
```

Comment:
minor thing, but I guess this metrics is kind of redundant, because we can get the same information from the existing ADD_MEDIA_FAILURE metric when it's sent with retriedWithTurnServer=true

Reply:
Removed these metrics